src/App.js

```diff
@@ -1,5 +1,7 @@
 import './App.css';
 import { Routes, Route } from 'react-router-dom';
```

Comment:
- [ ] I'm unable to install packages or run and test the app. Here is a screenshot of the error I'm getting on my system
![Screenshot from 2022-08-27 13-07-13](https://user-images.githubusercontent.com/46388113/187031806-ab33b603-9f0f-46b3-9e2f-43ec8932a07e.png)

Reply:
alright let me check since it was running properlyon mine

Reply:
Kindly do that and if possible add a screenshot of the UI

Reply:
try deleting the package.lock.jason and running it again